group_actions.py

```diff
@@ -101,6 +104,82 @@ def whitelist_package_in_group_devices(package_name):
             print('Warning: Package already whitelisted or not present on device: {}'
                 .format(device.device_name))
 
+# ====== INSTALL TO GROUP ==========
+
+
+"""
+Return application id of specific package name and version code stored 
+in environemtn. Return None if not present.
+"""
+def get_app_id(package_name, version_code):
+    app_api_instance = esperclient.ApplicationApi(esperclient.ApiClient(CONFIGURATION))
+    response = app_api_instance.get_all_applications(ENTERPRISE_ID)
+    #print(response)
+    i = 0
+
+    while (i < response.count):
+        # found the package already present in the environment
+        if response.results[i].package_name == package_name:
```

Comment:
check length of results. this will throw exception if results is empty list.

Reply:
length check is not required since we are already checking response.count above. but i have added null check for response.results exist.